apps/wallet/src/ui/app/shared/card/item/index.tsx

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { Text } from '_app/shared/text';
+
+import type { ReactNode } from 'react';
+
+export type CardItemProps = {
+    title: ReactNode;
+    value: ReactNode;
+};
+
+export function CardItem({ title, value }: CardItemProps) {
```

Comment:
FWIW I'd move these out of their own folders and just have files alongside the card like `/card/CardItem.tsx`. We should never have folders with just one file in them.